src/lnav_commands.cc

```diff
@@ -1063,6 +1063,64 @@ static string com_enable_filter(string cmdline, vector<string> &args)
     return retval;
 }
 
+static string com_enable_filters(string cmdline, vector<string> &args)
+{
+    if (args.size() == 0) {
+        args.push_back("disabled-filters");
```

Comment:
Since this command doesn't seem to take arguments, the args list should be left empty.  The command is called with zero args to figure out the type of arguments for doing tab-completion (not a great design, I know).